learn2track/models/ffnn.py

```diff
@@ -0,0 +1,126 @@
+from collections import OrderedDict
+from os.path import join as pjoin
+
+import numpy as np
+import smartlearner.initializers as initer
+import theano
+import theano.tensor as T
+from learn2track.models.layers import LayerDense
+from smartlearner import utils as smartutils
+from smartlearner.interfaces import Model
+
+floatX = theano.config.floatX
+
+
+class FFNN(Model):
+    """ A standard FFNN model with no output layer (using a sigmoid act. functions for the hidden layers).
+
+    The output is simply the state of the last hidden layer.
+    """
+
+    def __init__(self, input_size, hidden_sizes):
```

Comment:
We should support an additional hyperparmeter : the activation function. I'll would be interested in trying the ReLU